src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <h1 hidden>Calendar</h1>
+
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
```

Comment:
```suggestion
    <div class="calendar calendar--sun calendar--29">
```
можно классы вот так прописывать